pkg/policy/policiesSchema.json

```diff
@@ -0,0 +1,78 @@
+{
+  "type": "object",
+  "properties": {
+    "apiVersion": {
+      "type": "string",
+      "enum": [
+        "v1"
+      ]
+    },
+    "message": {
+      "type": "string",
+      "enum": [
+        "Should have exact one policy set as default",
+        "All policy names should be unique",
+        "All rule identifiers should be unique"
+      ]
+    },
+    "policies": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "required": [
+          "name"
+        ],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "isDefault": {
+            "type": "boolean"
+          },
+          "message": {
+            "type": "string"
+          },
+          "rules": {
+            "items": {
+              "type": "object",
+              "properties": {
+                "identifier": {
+                  "type": "string",
+                  "pattern": "^([A-Z0-9])+(_?[A-Z0-9])*$"
+                }
+              },
+              "required": [
+                "identifier",
+                "messageOnFailure"
+              ]
+            }
+          }
+        }
+      }
+    },
+    "customRules": {
+      "type": "array",
+      "items": {
+        "identifier": "string",
+        "pattern": "^([A-Z0-9])+(_?[A-Z0-9])*$",
```

Comment:
incorrect regex, this is what we have in backend:
![Screen Shot 2022-03-16 at 14 48 04](https://user-images.githubusercontent.com/39004075/158593229-0289ff30-ef25-4d1f-832c-f276b6ddd054.png)